dotnet/src/Connectors/Connectors.AI.HuggingFace/TextCompletion/HuggingFaceTextCompletion.cs

```diff
@@ -27,7 +27,7 @@ public sealed class HuggingFaceTextCompletion : ITextCompletion
     private readonly string? _endpoint;
     private readonly HttpClient _httpClient;
     private readonly string? _apiKey;
-    private readonly Dictionary<string, string> _attributes = new();
+    private readonly Dictionary<string, object> _attributes = new();
```

Comment:
Will the attributes of a service be immutable? 

If yes, we can store the values directly in the Attributes property from the ctor and no need to keep a Dictionary as private field.